gcs/folder.go

```diff
@@ -84,7 +103,11 @@ func (folder *Folder) ListFolder() (objects []storage.Object, subFolders []stora
 			return nil, nil, NewError(err, "Unable to iterate %v", folder.path)
 		}
 		if objAttrs.Prefix != "" {
-			subFolders = append(subFolders, NewFolder(folder.bucket, objAttrs.Prefix, folder.contextTimeout))
+
+			if objAttrs.Prefix != prefix+"/" {
```

Comment:
Missed spaces around operator "+"

Reply:
That's how automatic formatter is doing this for me...